.drone.yml

```diff
@@ -1,75 +1,100 @@
+---
+kind: pipeline
+name: testing
+
+platform:
+  os: linux
+  arch: amd64
+
 workspace:
   base: /go
   path: src/code.gitea.io/gitea
 
-pipeline:
-  fetch-tags:
-    image: docker:git
-    commands:
-      - git fetch --tags --force
+services:
+  - name: mysql
+    pull: default
+    image: mysql:5.7
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: yes
+      MYSQL_DATABASE: test
     when:
       event:
-        exclude: [ pull_request ]
+        - push
+        - tag
+        - pull_request
 
```

Comment:
remove the `when` condition for all services?

Reply:
Oh I totally overlooked that, thanks!
Fixed it.